tests/README.md

```diff
@@ -57,11 +59,7 @@ must `init` the submodules (if you did not `clone` with `--recursive`) and compi
 servers code:
 ```
 # From the root of the repository
-git submodule update --init --recursive
-cd tests/servers
-cd coordinatord && cargo build && cd ..
-cd cosignerd && cargo build && cd ..
-cd miradord && cargo build && cd ..
+./contrib/recompile_tests_servers.sh
```

Comment:
Just below this line it's stated:
> When you need a new version of the servers, you can update the submodules

Can you update this as well saying that they can just reuse the same script to update and recompile the tests servers?